### ISSUE TITLE
Can we remove core-icons.html fixed dependency?

### ISSUE THREAD

**Issue by pflannery** [Polymer/core-icon-button#11]
Would be nice to use this component without the 50kb or so overhead.

**Comment by sjmiles** [comment]
Yes, I agree with you. We really tried to structure things so that you could bring whatever icons to the table. 

I think the only concern is that somebody who has not read the documentation might be at a loss when they can't make `core-icon-*` actually produce an icon (because they have not imported any). But I think we need to solve this problem some other way than forcing a non-trival icon set.

**Comment by pflannery** [comment]
true, maybe a clear error message when no icon is found in the core-icon would help peeps?
